.drone.yml

```diff
@@ -51,12 +51,6 @@ pipeline:
     commands:
       - ./vendor/bin/phpunit
 
-  behat:
```

Comment:
Why you remove this step?

Reply:
It's in the PR description: "Removed behat, since drupal contribs are tested by functional test rather than behat." 
In this module no behat test will be provided.

Reply:
ok, thanks.